Cargo.toml

```diff
@@ -10,8 +10,6 @@ documentation = "http://ironframework.io/doc/router/index.html"
 keywords = ["iron", "web", "http", "routing", "router"]
 
 [dependencies]
+iron = "~0"
```

Comment:
>If we do have a new iron breaks current API, in 0.4 for example, we can then release a newer router

The thing is that the user pins both `iron` and `router`, so we can't just release a new version.

`0.x` releases always include breaking changes: In the case of `iron=0.3.0`, it was the hyper crate whose content we re-exported.

Anyway, this is not the appropriate place to discuss: Cargo expects us to pin by semver-compatible version, so pinning to `~0` is not acceptable.